index.php

```diff
@@ -2,31 +2,34 @@
 <html lang="en">
 <?php
 
+date_default_timezone_set("America/New_York");
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+$app = new Silex\Application();
+
+$app['debug']=true;
+
 use ABS\Classes\ModelClass as ModelClass;
 use ABS\Classes\ControllerClass as ControllerClass;
-use ABS\Classes\FooterViewClass as FooterViewClass;
-use ABS\Classes\HeaderViewClass as HeaderViewClass;
-use ABS\Classes\HeadViewClass as HeadViewClass;
-use ABS\Classes\MainViewClass as MainViewClass;
-
-require_once __DIR__.'/app/start.php';
 
 $model = new ModelClass();
-$headView = new HeadViewClass($model);
-$headerView = new HeaderViewClass($model);
-$mainView= new MainViewClass($model);
-$footerView = new FooterViewClass($model);
 $controller= new ControllerClass($model);
 
 if (isset($_GET['action']))
 {
 	$controller->{$_GET['action']}();
 }
 
-echo $headView->output();
-echo $headerView->output();
-echo $mainView->output($model->page);
-echo $footerView->output();
+$app->register(new Silex\Provider\TwigServiceProvider(), array( 
+		'twig.path' => __DIR__.'/views',
+	));
+
+$app->get('/', function () use ($app, $model) {
```

Comment:
you are currently using `$_GET['action']` to define which page your are hitting.

the idiomatic way of doing this on Silex is to have a different route for every page

so you would have something like

```
$app->get('/', function () use ($app) {
  return $app['twig']->render('home.twig');
});

$app->get('/about', function () use ($app) {
  return $app['twig']->render('about.twig');
});

$app->get('/contact', function () use ($app) {
  return $app['twig']->render('contact.twig');
});
```

and they would work on the urls <base-url>/about instead of <base-url>/?action=about

this should work fine if you are testing it on the PHP Built-in server (e.g. `php -S localhost:8080`)

for your server, you will likely need some extra configuration for this to work (see https://silex.symfony.com/doc/2.0/web_servers.html)